model/src/lib.rs

```diff
@@ -6,7 +6,10 @@ pub mod h160_hexadecimal;
 pub mod u256_decimal;
 
 use chrono::{offset::Utc, DateTime, NaiveDateTime};
+use ethcontract::common::abi::{encode, Token};
```

Comment:
Does model really have to depend on ethcontracts? Are encode and Token not reexported from some more basic type crate?